css/css-ui-3/text-overflow-012.html

```diff
@@ -0,0 +1,75 @@
+﻿<!DOCTYPE html>
+<html class="reftest-wait">
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: ellipsis and grapheme cluster</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#text-overflow">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem should">
+<!-- This is SHOULD because although CSS-UI defines "character" as "grapheme cluster" in a non optional manner
+  UAX29 itself (which defines grapheme cluster) only provides a baseline mechnism for determining grapheme clusters,
+  which implementations are expected to tailor and tweak. -->
```

Comment:
The explanation why this is a should wasn't very good, so I've provided a better one, but I still think we can only justify this as a should level. It would be nice to be stricter, but I don't think the current state of the UAX29 justify that.